about.html

```diff
@@ -0,0 +1,157 @@
+              <a href="#facebook"></a><i class="fa-brands fa-facebook-f"></i>
+            </li>
+            <li><a href="#twitter"></a><i class="fa-brands fa-twitter"></i></li>
+            <li><a href="#english"></a>English</li>
+            <li><a href="#myPage"></a>My Page</li>
+            <li><a href="#logout"></a>Logout</li>
+          </ul>
+        </div>
+      </div>
+      <div class="header-nav">
+        <div class="navbar">
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+          <div class="logo">
+            <a href="index.html"
+              ><img src="./assets/images/logo.PNG" alt="merctech mentorship"
+            /></a>
+          </div>
+          <nav class="nav-menu">
+            <ul class="menu-links">
+              <li>
+                <a href="index.html" class="a-links">Home</a>
+              </li>
+              <li>
+                <a href="#program" class="a-links">Program</a>
+              </li>
+              <li>
+                <a href="#join" class="a-links">Join</a>
+              </li>
+              <li>
+                <a href="#sponsor" class="a-links">Sponsor</a>
+              </li>
+              <li>
+                <a href="#news" class="a-links">News</a>
+              </li>
+              <li class="campaign">
+                <a href="#campaign" class="a-links">campaign</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <div class="header-contents-about">
+      <div class="container-herooo">
+        <div class="landing-text-about">
+          <div>
+            <img
+              src="./assets/images/theme1.PNG"
+              class="hero-img"
+              alt="theme"
+            />
+          </div>
+          <div class="about-text">
+            <p class="text-about">
+              Potential is equally distributed; opportunity is not. MENTOR aims to drive equity 
+              and close the mentoring gap through quality mentoring relationships for young people.
+              We activate a diverse cross-sector movement that prioritizes 
+              relationships 
+            </p>
+          </div>
+          <div class="contact-about">
+            <p>
+              Please contact us per Email for nay further questions about Bridge gap mentorship.
+            </p>
+            <p class="email">bridgegap@mentoring.org</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    <section class="logo-info">
+        <p class="creative-logo">Bridge the Gap mentorship logo</p>
+        <p class="title-underline"></p>
+
+        <p class="logocc">
+          The logo of Bridge the Gap mentorship was decided through the logo
+          competition from 8, nov to 7, dec.
+        </p>
+        <div class="aboutlg">
+          <img src="./assets/images/about_logo 1.png" />
+        </div>
```

Comment:
Nice work but for a better design, and more fitting the rest of the page 🎨 consider using a `border` for this, just like below 👇🏽 

![Screenshot 2023-03-18 at 14 46 13](https://user-images.githubusercontent.com/45739963/226129899-216c0637-aab6-4696-8d95-645af8d26a38.png)